pkg/storage/mock/storer.go

```diff
@@ -263,6 +263,9 @@ func (m *MockStorer) SubscribePush(ctx context.Context) (c <-chan swarm.Chunk, s
 func (m *MockStorer) PinnedChunks(ctx context.Context, cursor swarm.Address) (pinnedChunks []*storage.Pinner, err error) {
 	m.pinSetMu.Lock()
 	defer m.pinSetMu.Unlock()
+	if m.pinnedAddress == nil || len(m.pinnedAddress) == 0 {
```

Comment:
it is actually not necessary to check whether `m.pinnedAddress` is `nil`, since `len()` of uninitialized slice [will return zero anyway](https://play.golang.org/p/wTsFgdfr_qx)